openshift4/tusd/base/deployment.tusd.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
+kind: Deployment
+metadata:
+  name: tusd
+parameters:
```

Comment:
I dont think deployments can take parameters, as that's a template behaviour. I'm guessing you kept this because you're thinking of which tag for dev/test/prod? I think this is where we can take advantage of kustomize. I'd hardcode it to `tusd:dev` for the base, then use the kustomize patches to overwrite for `tusd:test` and `tusd:prod`